### PR TITLE
chore(weave): Fix FE Devmode

### PR DESCRIPTION
**Pull request by tssweeney**

### CODE REVIEW

File: weave-js/vite.config.ts [hunk 1]
```diff
@@ -138,7 +138,14 @@ export default defineConfig(({mode, command}) => {
           secure: false,
           changeOrigin: true,
         },
-        '^.*/__frontend/.*': {
+        // Dynamically generated env.js
+        '^.*/__frontend/env.js': {
```
Comment (vwrj) [inline weave-js/vite.config.ts:142]: Wouldn't this be captured by the regex you inserted?

Comment (vwrj) [inline weave-js/vite.config.ts:142]: Wouldn't this be captured by the regex you inserted below?

**Review by vwrj — APPROVED** [review]

File: weave-js/vite.config.ts (hunk 1 above)
Comment (tssweeney) [reply weave-js/vite.config.ts:142]: it would be, but i kind of like the explicit nature of env as it does not exist on disk

Comment (vwrj) [reply weave-js/vite.config.ts:142]: sg

**Pull request merged**